### PR TITLE
updated web3

### PR DESCRIPTION
**Pull request by 0xModene**
# **Pull Request Template**

###### *Select a category that relates to the content of your pull request (choose all that apply).*

- [x] Bug Fix
- [ ] Content
- [ ] New Feature
- [ ] Documentation
- [ ] Code Refactoring

&nbsp;

## **Summary of Changes**
updating dependencies to help solve trezor issues

&nbsp;

### **Fixes: Issue #**

&nbsp;

###### *Include proof that your changes function properly and resolve the linked issue.*

&nbsp;

###### *By submitting this pull request, you are confirming the following to be true:*

- I have reviewed the [Contribution Guidelines](https://github.com/SetProtocol/index-ui/blob/master/CONTRIBUTING.md).
- I have performed a self-review of my own code.
- I have updated my repository to match the master at `SetProtocol/index-ui`.
- I have included test data or screenshots that prove my fix is effective or that my feature works.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "set.js": "^0.3.2",
     "styled-components": "^5.3.3",
     "use-wallet": "^0.11.0",
-    "web3": "^1.5.2",
+    "web3": "^1.6.0",
```
Comment (rootulp) [inline package.json:39]: LGTM! 

[Off topic] Is there any interest in switching from web3 to ethers which has much stronger TypeScript support?

**Review by rootulp — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (0xModene) [reply package.json:39]: For me personally? Yes, I'd like to on the rewrite

Comment (controtie) [reply package.json:39]: Same, I'm a big fan of others

**Review by controtie — APPROVED** [review]
LGTM

**Pull request merged**